tsinfer/inference.py

```diff
@@ -469,7 +469,16 @@ def add_sites(self):
         logger.info("Starting addition of {} sites".format(self.num_sites))
         progress = self.progress_monitor.get("ga_add_sites", self.num_sites)
         for j, variant in enumerate(self.sample_data.variants(inference_sites=True)):
-            self.ancestor_builder.add_site(j, variant.site.time, variant.genotypes)
+            time = variant.site.time
+            if time == self.sample_data.TIME_UNSPECIFIED:
```

Comment:
Would be better to make this a global constance, like ``tsinfer.TIME_UNSPECIFIED`` rather than a class constant.